src/main/java/com/gmail/yuramitryahin/dao/MovieSessionDao.java

```diff
@@ -12,4 +12,6 @@
     MovieSession update(MovieSession movieSession);
 
     void delete(Long id);
+
+    MovieSession get(Long id);
```

Comment:
should we return Optional here?

Reply:
fixed

Reply:
but if we do not find such an object, will we still have an exception?

Reply:
sure - will we have an exception if we try to get value from empty Optional on the service level

Reply:
ok